packages/react-components/react-progress/src/components/Progress/Progress.types.ts

```diff
@@ -2,45 +2,31 @@ import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utili
 
 export type ProgressSlots = {
   /**
-   * The root of the Progress
-   * The root slot receives the `className` and `style` specified directly on the `<Progress>`.
+   * The track behind the progress bar
    */
   root: NonNullable<Slot<'div'>>;
   /**
-   * The title of the Progress.
-   * The label slot receives the styling related to the title associated with the Progress.
-   */
-  label?: Slot<'span'>;
-  /**
-   * The animated slot of the Progress
-   * The bar slot receives the styling related to the loading bar associated with the Progress
+   * The filled portion of the progress bar. Animated in the indeterminate state, when no value is provided.
    */
   bar?: NonNullable<Slot<'div'>>;
-  /**
-   * The track slot of the Progress
-   * The track slot receives the styling related to the loading bar track associated with the Progress
-   */
-  track?: NonNullable<Slot<'div'>>;
-  /**
-   * The description slot of the Progress
-   * The description slot receives the styling related to the description associated with the Progress
-   */
-  description?: Slot<'span'>;
 };
 
 /**
  * Progress Props
  */
 export type ProgressProps = Omit<ComponentProps<ProgressSlots>, 'size'> & {
   /**
-   * Prop to set whether the Progress is determinate or indeterminate
-   * @default false
+   * A decimal number between `0` and `1` (or between `0` and `max` if given),
+   * which specifies how much of the task has been completed. 
+   *
+   * If `undefined` (default), the Progress will display an **indeterminate** state.
    */
-  indeterminate?: boolean;
+  value?: number;
   /**
-   * Percentage of the operation's completeness, numerically between 0 and 100.
+   * Max value of the determinate progress.
```

Comment:
nit: another place to remove 'determinate' from the documentation. How about this?

```suggestion
   * The maximum value, which indicates the task is complete.
   * The progress bar will be full when `value` equals `max`.
```